app/views/reports/_auditing_report.html.haml

```diff
@@ -22,12 +22,12 @@
   .modal-content
     = form_tag reports_path, remote: true, class: "form-horizontal", role: "form" do
       .modal-header
+        %h4.modal-title.text-center
+          = t(:reports)[:auditing_report]
         %button.close{type: 'button', data: {dismiss: 'modal'}}
           %span{aria: {hidden:'true'}} &times;
           %span.sr-only
             = t(:actions)[:close]
-        %h4.modal-title.text-center
-          = t(:reports)[:auditing_report]
       .modal-body
         #modal_errors
```

Comment:
Can you replace the `#modal_errors` content with inline error messages? Here's an example of it in SPARC: https://github.com/sparc-request/sparc-request/blob/v3.9.0/app/views/arms/create.js.coffee

Reply:
I think the error messages work fine for now like this, and I think it's out of scope for bootstrap to change now. Will put stories in for next release.